src/centos/6/Dockerfile

```diff
@@ -4,20 +4,22 @@ FROM centos:6
 
 RUN yum install -y \
         centos-release-SCL \
+        scl-utils-build \
```

Comment:
Can this be alphabetized?  Same comment applies in other places as well.

Reply:
Done